src/ML-PACE/compute_pace.h

```diff
@@ -0,0 +1,86 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
```

Comment:
The contact email should be:
```
LAMMPS development team: developers@lammps.org
```
It should be possible to update this with `make fix-homepage`
Also, please run `make fix-whitespace` in the src folder and commit those changes as well to have consistent whitespace (no tabs, no trailing whitespace). If you want to go one step further, you could also run clang-format on both, the header and the implementation.